suite-native/module-send/src/components/SendForm.tsx

```diff
@@ -48,11 +62,45 @@ export const SendForm = ({ accountKey }: SendFormProps) => {
 
     const {
         handleSubmit,
-        formState: { isValid },
+        watch,
+        formState: { isValid: isFormValid },
     } = form;
+    const rawValues = watch();
+
+    useEffect(() => {
```

Comment:
Why is this an effect? You could debounce even in onChange handler and could be more deterministic...